install_config/configuring_aws.adoc

```diff
@@ -276,10 +227,17 @@ key `kubernetes.io/cluster/<name>,Value=<clusterid>` and have the same value for
 $ oc get pv -o json|jq '.items[].spec.awsElasticBlockStore.volumeID'
 ----
 
-. Restart the master services on the master and the node service on all nodes:
+. Restart the master services on the master and the node service on all nodes,  see xref:../../install_config/master_node_configuration#master-node-config-restart-services[Restarting {product-title} services]:
 +
-[source,bash]
+ifdef::openshift-enterprise[]
```

Comment:
The above line already link user to 'Master and Node Configuration' section, why here are still these command lines for serverice restart. Should we remove them?